x-pack/solutions/search/plugins/enterprise_search/public/navigation_tree.ts

```diff
@@ -300,7 +300,6 @@ export const getNavigationTreeDefinition = ({
                     },
                   ],
                   id: 'stack_management', // This id can't be changed as we use it to open the panel programmatically
-                  link: 'management',
```

Comment:
This removes the "four squares" icon from the nav item and creates a single click target, which has an icon of an arrow pointing to the right.